pixi.toml

```
@@ -0,0 +1,37 @@
+[project]
+name = "multi-rate-mpc-env"
+channels = ["conda-forge"]
+platforms = ["linux-64", "linux-aarch64", "osx-arm64", "win-64"]
+
+[activation.env]
+CMAKE_INSTALL_PREFIX = "$CONDA_PREFIX"
+
+[target.win-64.activation.env]
+CMAKE_INSTALL_PREFIX = "%CONDA_PREFIX%\\Library"
+
+[dependencies]
+compilers = "*"
```

As we do not use a fortran compiler, we can just depend on `cxx-compiler`:

```suggestion
cxx-compiler = "*"
```